src/measure_theory/decomposition/jordan.lean

```diff
@@ -420,6 +428,46 @@ begin
     rw [total_variation, measure.add_apply, h.1 hS₂, h.2 hS₂, add_zero] }
 end
 
+lemma mutually_singular_iff (s t : signed_measure α) :
+  s ⊥ᵥ t ↔ s.total_variation ⊥ₘ t.total_variation :=
```

Comment:
Surely we want the generalization of this lemma where `s` and `t` are arbitrary vector measures (with potentially different codomains).

Presumably you can get this by showing that a vector measure is concentrated on some set A iff its total variation is concentrated on A.

Reply:
We don't have total variation on vector measures just yet, I will add it in another pr

Reply:
Ok! Perhaps you could add a comment saying that this will generalize when total variation of vector measures is defined, just so we don't lose track.